src/mouse_events.cpp

```diff
@@ -349,6 +362,17 @@ unit_map::const_iterator mouse_handler::find_unit(const map_location& hex) const
 	return pc_.gamestate().board_.find_visible_unit(hex, viewing_team());
 }
 
+/**  Wraps 'const map_location game_display::hex_clicked_on(int, int)
```

Comment:
Doxygen comments like these should be with the method declaration in the header, using this format:
```
/**
 * Text starts here.
 */
```
The text of this comment should read: `Const version of @ref hex_clicked_on.`. Doxygen will point to that function if you use `@ref`.

Reply:
Oh, I missed that. Yeah. doxy-comments should definitely be in the header.

Reply:
Will comply request.